pkg/cvo/upgradeable.go

```diff
@@ -263,6 +274,45 @@ func (check *clusterVersionOverridesUpgradeable) Check() *configv1.ClusterOperat
 	return cond
 }
 
+type upgradeInProgressUpgradeable struct {
+	name     string
+	cvLister configlistersv1.ClusterVersionLister
+}
+
+func (check *upgradeInProgressUpgradeable) Check() *configv1.ClusterOperatorStatusCondition {
+	cond := &configv1.ClusterOperatorStatusCondition{
+		Type:   configv1.OperatorProgressing,
```

Comment:
Does `Type=configv1.OperatorProgressing` confuse the `collectUpgradeableConditions` function?


https://github.com/openshift/cluster-version-operator/blob/3f49cddaf1d2dc58cb07e6a072909d9392b33554/pkg/cvo/upgradeable.go#L160